### ISSUE TITLE
App requires admob to run

### ISSUE THREAD

**Issue by SiavashB** [ernestoyaquello/KeepieUppieGame#1]
Hi, this repo as it is cannot be run, I even created a dummy admob.properties file to get passed the build errors but the app crashes at run time since I don't have the proper admit ids. Would be nice to create a build variant that doesn't use adds and thus can be ran by people trying to learn through this repo. 

Looking at the code is very useful but, this could be much more educational if I can run it in debug mode.

Thank you.

**Comment by ernestoyaquello** [comment]
You are right, but to be honest, this code is here as an example/code dump, so I didn't expect anyone to run it. And I would rather keep it as it is, as I might need to use it in the future.

If you are really interested in running it, just remove all the references to AdMob. If you aren't able to remove them all or find other problems when trying to run it, please let me know and I will try to help.
